### ISSUE TITLE
No rule...

### ISSUE THREAD

**Issue by wwderw** [thinkuped/thred#1]
When trying to make, I get the following back:

 *** No rule to make target '/usr/include/wine/windows/tchar.h', needed by 'tchar/h'.

Any ideas on how to fix that?

Thanks.

**Comment by thinkuped** [comment]
You need to be running WINE:
https://www.linux.com/learn/how-install-and-use-wine-run-windows-applications-linux

**Comment by wwderw** [comment]
The distro that I use already has WINE up and running on it from default.  I have been using the original Thred Windows files through WINE, but I was hoping to build this and create an appimage to run as a stand alone.

**Comment by thinkuped** [comment]
Sorry, I just imported this code from the original Thred project and
haven't done anything with it yet.

**Comment by margaretlange** [comment]
Hey, is thred being actively maintained these days?

**Comment by wwderw** [comment]
@margaretlange I don't think so.  Last commit was in 2012 and it's still dependent on WINE to run in Linux.

There is another project that I'm contributing to that is an embroidery extension for Inkscape (it works on Linux and Windows natively, no Mac support at this time, although that is being worked on) that is more feature complete then Thred.  It's called inkstitch.

May want to give that a look see if you are looking for something for digitizing.

**Comment by margaretlange** [comment]
thank you!
